src/User.js

```diff
@@ -36,16 +36,45 @@ class User {
     return dailyOunces.numOunces;
   };
 
-  returnUserWeekData(array) {
+  returnUserWeekData(array, neededData) {
     const userArray = this.findUser(array);
     const userDatesArray = userArray
       .splice(0,7)
-      .map(hydrationObj => hydrationObj.numOunces)
```

Comment:
We should probably rename this variable because it's no longer a hydrationObj - maybe pieceOfData?

Reply:
- I like renaming the variable since it pertains to several different items; let's make that change and it will help with dev empathy as well.